### PR TITLE
Adds validate call to rl_runner. Fixes #43

### LINKED ISSUE #43: Create validation set (self-play and professional)
For some small percent of self-play games, we should set them aside, never train on them, and check the value net loss on these games. This should help us figure out if we're overfitting.

A variant on this is to create a validation set of pro games. Self-play validation sets are nice because there's not a very large library of pro 9x9 games. But for 19x19, a pro validation set is doable.

### PR DESCRIPTION
**Pull request by amj**
Also:
- Changes default # of validation steps 
- Ensures the summary log is written afterwards

Fixes #43

### CODE REVIEW

File: rl_runner.py [hunk 1]
```diff
@@ -44,6 +44,10 @@ def loop(logdir=None):
             subprocess.call(
                 ("python rl_loop.py train --logdir=%s" % logdir).split())
 
+        with timer("validate"):
```
Comment (Kashomon) [inline rl_runner.py]: Hm. Do we want the trainer to do both validation and training?

Comment (amj) [reply rl_runner.py]: yeah, why not?  validate is ~a minute, or about ~1% of overhead

Comment (Kashomon) [reply rl_runner.py]: SGTM.

Comment (amj) [reply rl_runner.py]: We might re-visit this when we containerize the trainer; Brian mentioned that he'd like more validation steps, which would move us from 1% overhead up to 2% or 3% which seems like enough to break validation off of the main trainer process.
